compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformContext.skiko.kt

```diff
@@ -125,6 +126,15 @@ interface PlatformContext {
          * @see Owner.onSemanticsChange
          */
         fun onSemanticsChange(semanticsOwner: SemanticsOwner)
+
+        /**
+         * Callback method that is called when the position and/or size of the [LayoutNode] with
+         * the given semantics id changed.
+         *
+         * Note that the id, rather than the [LayoutNode] itself, is passed here because
+         * [LayoutNode] is an internal type, so it can't be exposed in a public method.
+         */
+        fun onLayoutChange(semanticsOwner: SemanticsOwner, semanticsNodeId: Int)
```

Comment:
We already discussed this with @elijah-semyonov and as far as I remember we decided to introduce optional second parameter to `onSemanticsChange` instead of adding a new callback

Reply:
Having it in a separate function adds more information; namely that only the bounds of the node have changed, and not any semantic properties.

Reply:
The API changed here isn't public, by the way, so no need for the "changed in API" tag.

Reply:
it's public but with "internal" annotation. The rule for tag is basically - "changes in `desktop/ui.api` file".
I'd apply the rule regarding two reviewers here

Reply:
I disagree. @InternalApi implies that it's internal, and only technically public because of the limitations of the language. We don't make any compatibility guarantees about such code.

Reply:
ok, it's not a big deal in this case